docs/user-guide.md

```diff
@@ -489,6 +516,38 @@ The partner can only generate a Draft Report after running `Compliance Collectio
 
 ![](\_images/ctk-report-submitted.png)
 
+### Generate Quality Assessment Draft Report for Quality Assessment Collection
+
+The partner can only generate a QA Draft Report after running `Quality Assessment Collection`.
+
+* By default, `Quality Assessment Collection` will be added when the SBI project was created.
+
+![](\_images/ctk-qa-collection-created.png)
+
+* Click the `Run` button of the Quality Assessment Collection.
+
+![](\_images/ctk-qa-collection-run.png)
+
+* After completing the test execution, the partner will be able to view the below mentioned results
+  * A high-level result with the number of test cases executed- with the number of test cases failed and the number of test cases passed in the run.
```

Comment:
A summary of the test execution results includes the total number of test cases executed, the count of test cases that failed, and the number of test cases that passed during the run.